### ISSUE TITLE
Buttons are not labeled when using talkback on watch.

### ISSUE THREAD

**Issue by CatsAreCool710** [bryan2894-playgrnd/SimpleWear#60]
When using talkback on the watch, buttons are not labeled. Please consider adding accessibility labels on the phone and watch apps.

**Comment by thewizrd** [comment]
Accessibility labels should be improved as of v1.16.0. If any are missing let me know
